### PR TITLE
Populates TensorProto with tensor from protobuf file

### PR DESCRIPTION
**Pull request by ryanlai2**
This change exposes this function : LoadOnnxTensorFromProtobufFile in protobufHelpers.h which allows tests to populate TensorProto with tensor from protobuf file.

### CODE REVIEW

**Comment by martinb35** [comment]
are you actually calling it from a new place? look like you just renamed it and put it in the header. or is the new call coming later?

**Comment by ryanlai2** [comment]
> are you actually calling it from a new place? look like you just renamed it and put it in the header. or is the new call coming later?

Yep, I renamed it and put it in the header. I am calling it in internal performance tests that require this function to extract the tensor name from TensorProto. The new call will be in internal code, not in a separate PR on github.

**Comment by snnn** [comment]
Maybe it is out of topic, but I highly suggest please do not use C++ fstream in our code, because they're too slow on Azure VMs. 

Our Azure build VM has a IOPS cap of 500, which is far less a physical SSD can do.
https://azure.microsoft.com/en-us/pricing/details/managed-disks/

A real SSD can do 100k-500k reads per second!
https://www.intel.com/content/dam/www/public/us/en/documents/product-specifications/ssd-750-spec.pdf

With 500 IOPS and 4KB buffer size, we can only achieve 2MB/s disk speed, which means it need about 5 minutes to load an ONNX model zoo model, 4 hours to read all of our test data, which is not acceptable.  We can't let customers know onnxruntime is so slow. 

The most ugly part is: C++ fstream does not allow you change the buffer size.

**Comment by ryanlai2** [comment]
> Maybe it is out of topic, but I highly suggest please do not use C++ fstream in our code, because they're too slow on Azure VMs.
> 
> Our Azure build VM has a IOPS cap of 500, which is far less a physical SSD can do.
> https://azure.microsoft.com/en-us/pricing/details/managed-disks/
> 
> A real SSD can do 100k-500k reads per second!
> https://www.intel.com/content/dam/www/public/us/en/documents/product-specifications/ssd-750-spec.pdf
> 
> With 500 IOPS and 4KB buffer size, we can only achieve 2MB/s disk speed, which means it need about 5 minutes to load an ONNX model zoo model, 4 hours to read all of our test data, which is not acceptable. We can't let customers know onnxruntime is so slow.
> 
> The most ugly part is: C++ fstream does not allow you change the buffer size.

Thanks for the suggestion Changming. I've updated the PR to use parsefromzerocopystream rather than use fstream!

File: winml/test/common/protobufHelpers.cpp [hunk 1]
```diff
@@ -44,25 +43,31 @@ void FdClose(int fd) {
...
+  int block_size = -1;
+  int fd;
+  // Get file descriptor
+  onnxruntime::Status status = onnxruntime::Env::Default().FileOpenRd(filePath, fd);
+  if (!status.IsOK()) {
+    return false;
+  }
+  // Get size of file
+  status = onnxruntime::Env::Default().GetFileLength(fd, file_size);
+  if (status.IsOK()) {
```
Comment (snnn) [inline winml/test/common/protobufHelpers.cpp:58]: Don't need be so complex. This is test code, you can directly set the block size 4M. The call of GetFileLength is just to save a few memory when the model size is small, which is not a concern for such a test program. 
It's up to you, you may keep it here.

Comment (ryanlai2) [reply winml/test/common/protobufHelpers.cpp:58]: Makes sense, thanks for the suggestion!

**Pull request merged**